dev-requirements.txt

```diff
@@ -21,8 +21,9 @@ async-timeout==4.0.2  # Used for timeouts in some test cases.
 # DOCUMENTATION #
 #################
 
-pdoc3==0.10.0
+pdoc==11.0.0
```

Comment:
Bump to 11.2.0

Things to do:

- [ ] Reduce heading font size within docstrings: https://github.com/mitmproxy/pdoc/pull/383
- [ ] Better admonitions: https://github.com/mitmproxy/pdoc/pull/372
- [ ] Think about https://github.com/mitmproxy/pdoc/pull/379
- [ ] Wait for release or implement https://github.com/mitmproxy/pdoc/pull/391


With all this said, maybe consider not making our own template, but rather using the default one with slight modifications only